ruby_on_rails/forms_and_authentication/project_forms.md

```diff
@@ -85,6 +111,20 @@ Now we'll start morphing our form into a full Rails form using the `#form_tag` a
 1. Test out your form.  You'll need to change your `#create` method in the controller to once again accept normal top level User attributes, so uncomment the old `User.new` line and comment out the newer one.
 1. You've just finished the first step.
 
+#### Turn Turbo back ON
+
+Above, we asked to disable Turbo for the sake of the exercise.
+
+1. Re-enable form submission with Turbo by removing the `data-turbo=false` attribute on the form tag, then also remove the hidden input with CSRF token tag and submit.
+
+   No more CSRF error!?!
+   The from is now submitted with Turbo, yet Rails still protects you by verifying a CSRF token. Where does this token comes from?
```

Comment:
I am not a fan of how these lines lay out in the markdown preview either, though I don't really have a great solution in mind. Maybe add an empty line between 120 and 121?

Reply:
Here I have added an empty line to keep the "No more CSRF error" as a standalone observation that the reader should pause on. I did not put it on a separate numbered list item as it s not a new task.